bip-0142.mediawiki

```diff
@@ -126,7 +126,7 @@ when encoded as a P2PKH template, would become:
   
     DUP HASH160 <010966776006953D5567439E5E39F86A0D273BEE> EQUALVERIFY CHECKSIG
 
-And the corresponding version 1 Bitcoin address is
+With the corresponding version 1 Bitcoin address being:
```

Comment:
It is called version 1 address on the Bitcoin wiki: https://en.bitcoin.it/wiki/Technical_background_of_version_1_Bitcoin_addresses